node/libs/crypto/src/keccak256/mod.rs

```diff
@@ -1,17 +1,18 @@
-//! Wrappers for the SHA256 cryptographic hash algorithm.
+//! Wrappers for the Keccak256 cryptographic hash algorithm.
 use crate::ByteFmt;
-use sha2::{digest::Update as _, Digest as _};
+use sha3::{digest::Update as _, Digest as _};
 
+mod test;
 pub mod testonly;
```

Comment:
Off-topic (?): Why does this module need to be public? AFAICT, it doesn't contain public items.

Reply:
Not sure if you refer to line 5 (added) or 6 (already exists). 
* `testonly`: it needs to be public, although it works without the pub modifier, because visibility is bound to the trait implementation target type. 
* `test`: I forgot to add the `#[cfg(test)]` attribute. Added now.